scripts/check_delta.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Get different tests delta between two xml files,
```

Comment:
Add sample command how to run that script in the description, or readme file. So any user can understand how to use it, without digging into details.
For example for be wasn't obvious that for each xml file which I provide, I should use -x option.
Example must have.

Reply:
added description header.